Source/Icebreaker/Helpers/TeamInstallInfo.cs

```diff
@@ -43,6 +44,12 @@ public string TeamId
         [JsonProperty("installerName")]
         public string InstallerName { get; set; }
 
+        /// <summary>
+        /// Gets or sets the ids of the feedback for this team
+        /// </summary>
+        [JsonProperty("feedbackIds")]
+        public List<string> FeedbackIds { get; set; }
```

Comment:
Is there a reason to store the ids like this in a separate collection rather than just storing a `List<UserFeedback>` directly in the `TeamInstallInfo`? 

Since icebreaker is using the document API for cosmos, it seems more natural to nest the data unless there is some case where the feedback will be accessed outside the scope of the team. 

Alternatively, would it be acceptable to omit this list & rely on the fact that each `UserFeedback` has a reference to the `TeamId` which is [just the id](https://github.com/OfficeDev/microsoft-teams-apps-icebreaker/blob/e5cb7704d25c7b18aaeebde6e3b9aac45b185206/Source/Icebreaker/Helpers/TeamInstallInfo.cs#L22) of this `TeamInstallInfo`?

Reply:
I just got rid of this field, I think it makes sense to store the feedback together in a separate collection rather than under `TeamInstallInfo` since the feedback is geared toward the app itself and doesn't affect matchings or individual teams that much. Also, like you said, each `UserFeedback` does have a reference to the `TeamId` so if someone wanted to pull feedback for an individual team, that would also be possible.